ulacitbnb/Controllers/PaymentController.cs

```diff
@@ -135,6 +135,7 @@ public IHttpActionResult Enter(Payment payment)
                     sqlCommand.Parameters.AddWithValue("@Pay_Total", payment.Pay_Total);
 
                     sqlConnection.Open();
+                    int rowsAffected = sqlCommand.ExecuteNonQuery();
```

Comment:
There is no need to create a variable, just do this

```suggestion
                    sqlCommand.ExecuteNonQuery();
```